lib/rule_event_repository_serializer.rb

```diff
@@ -0,0 +1,13 @@
+class RuleEventRepositorySerializer
+  class << self
+    def dump(repository)
+      repository.dump
+    end
+
+    def load(data)
+      return RuleEventRepository.new if data.nil?
```

Comment:
This is needed unless we do `t.json "events", default: '{"events": []}'` which would bleed ruby logic in mysql land.

Reply:
So let's do `default: '[]'`?

Reply:
We don't "need" it, but it is a nice separation of concerns.

> While verbose, and since already coded, I think it make sense to keep them separate. If you prefer I can make the change. Since this is only ruby and not related to the data model, I don't really mind either way.

I can add a commit that removes it and we'll see

Reply:
Commit that removes the serializer: https://github.com/christianblais/triggerify/pull/116/commits/8167a129ce525b789bc2d8e6ed80be95c8f32bdb